experiment.py

```diff
@@ -76,8 +77,7 @@ def __init__(self, entry):
         for cell in self.cells.values():
             for label in labels:
                 assert label in cell.labels
-            for cre in cre_types:
-                assert cre in cell.labels
+            assert cell.cre_type in cre_types
```

Comment:
originally I think it was to make sure that the cre_types and labels matched up since they were being pulled from different places, it might not be necessary now since we're constructing the cre_types from the cells as opposed to the labels?